### ISSUE TITLE
`$AU-color-error` not a11y

### ISSUE THREAD

**Issue by sukhrajghuman** [govau/design-system-components#656]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/design-system-components/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/design-system-components/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

The error color does not meet WCAG AA on both dark and light backgrounds. Will need to change this to meet WCAG AA
![image](https://user-images.githubusercontent.com/20184809/53450677-06203400-3a71-11e9-9a98-b200d8677589.png)

<hr />

![image](https://user-images.githubusercontent.com/20184809/53451282-8b581880-3a72-11e9-96b6-40f606d8b86a.png)

**Comment by sukhrajghuman** [comment]
closed by #690 

It's difficult to get an a11y red on dark background. Will have to provide guidance around using form controls on lighter backgrounds
